web/templates/user/user_show.html

```diff
@@ -30,9 +30,9 @@
                {{ 'aria-disabled="true"' if 'disabled' in tab and tab.disabled else "" }}
                id="tab-{{ tab.id }}" href="#{{ tab.id }}"
                data-bs-toggle="tab" data-bs-target="#{{ tab.id }}">
-              {{ tab.name }}
+              {{ tab.name | safe }}
```

Comment:
Adding a dedicated `icon` attribute to the tab object would be a better way in my opinion.

Reply:
good point. That was the quick solution at the beginning where I was unsure how the result would look like.